python/lsst/cp/pipe/ptc.py

```diff
@@ -108,6 +110,12 @@ class MeasurePhotonTransferCurveTaskConfig(pexConfig.Config):
         doc="Maximum value to of mean signal (in ADU) to consider.",
         default=9e6,
     )
+    initialNonLinearityExclusionThreshold = pexConfig.Field(
+        dtype=float,
+        doc="Initially exclude data points that are more than a factor of this from being linear from"
```

Comment:
Linear in the var vs mean curve, correct? Maybe a clarification is in order since the code also calculates linearity residuals from "mean vs time".

Reply:
Well, the mask is applied to both if you read the code; this initial cut is indeed just to the PTC code, but points that are thrown out of the PTC fit are also excluded from the non-linearity fit. I've updated the docstring to make that as clear as possible though.